This is needed to prevent pkgdepend to generate duplicate dependencies.

--- mypy-0.991/mypy/stubgen.py
+++ mypy-0.991/mypy/stubgen.py
@@ -1,4 +1,3 @@
-#!/usr/bin/env python3
 """Generator of dynamically typed draft stubs for arbitrary modules.
 
 The logic of this script can be split in three steps:
--- mypy-0.991/mypy/stubgenc.py
+++ mypy-0.991/mypy/stubgenc.py
@@ -1,4 +1,3 @@
-#!/usr/bin/env python3
 """Stub generator for C modules.
 
 The public interface is via the mypy.stubgen module.
